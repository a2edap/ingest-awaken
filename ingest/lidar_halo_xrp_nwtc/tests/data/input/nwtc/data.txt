# TODO â€“ Developer: Add your input test data here
Replace this file with a small data file for ingest development and testing. 

Note that you must ensure that you have permission to release the file because it will
be hosted publicly on github.