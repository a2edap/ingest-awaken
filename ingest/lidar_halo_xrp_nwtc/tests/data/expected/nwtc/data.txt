# TODO â€“ Developer: Add your expected output test data here
Place your expected output for the provided input file here. This will be used in
regular test cases, so again, it is best to keep this relatively small, if possible.